test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml

```diff
@@ -11,6 +11,7 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
 azure_instance_type_db: 'Standard_L16s_v3'
+gce_instance_type_db: 'n1-highmem-32'
```

Comment:
same here

Reply:
We wanted to test 24/32 cores for large instances that's why I put 32 here

Reply:
Changed to 16